lib/debezium/types.go

```diff
@@ -33,15 +33,15 @@ const (
 	EnumSet SupportedDebeziumType = "io.debezium.data.EnumSet"
 	UUID    SupportedDebeziumType = "io.debezium.data.Uuid"
 
-	Timestamp            SupportedDebeziumType = "io.debezium.time.Timestamp"
-	MicroTimestamp       SupportedDebeziumType = "io.debezium.time.MicroTimestamp"
-	NanoTimestamp        SupportedDebeziumType = "io.debezium.time.NanoTimestamp"
-	Date                 SupportedDebeziumType = "io.debezium.time.Date"
-	Year                 SupportedDebeziumType = "io.debezium.time.Year"
-	DateTimeWithTimezone SupportedDebeziumType = "io.debezium.time.ZonedTimestamp"
-	MicroDuration        SupportedDebeziumType = "io.debezium.time.MicroDuration"
-	DateKafkaConnect     SupportedDebeziumType = "org.apache.kafka.connect.data.Date"
-	DateTimeKafkaConnect SupportedDebeziumType = "org.apache.kafka.connect.data.Timestamp"
+	MicroTimestamp        SupportedDebeziumType = "io.debezium.time.MicroTimestamp"
+	NanoTimestamp         SupportedDebeziumType = "io.debezium.time.NanoTimestamp"
+	Date                  SupportedDebeziumType = "io.debezium.time.Date"
+	Year                  SupportedDebeziumType = "io.debezium.time.Year"
+	DateTimeWithTimezone  SupportedDebeziumType = "io.debezium.time.ZonedTimestamp"
+	MicroDuration         SupportedDebeziumType = "io.debezium.time.MicroDuration"
+	DateKafkaConnect      SupportedDebeziumType = "org.apache.kafka.connect.data.Date"
+	Timestamp             SupportedDebeziumType = "io.debezium.time.Timestamp"
+	TimestampKafkaConnect SupportedDebeziumType = "org.apache.kafka.connect.data.Timestamp"
```

Comment:
Only change here is that I renamed this to `TimestampKafkaConnect` to be more correct